tests/test_polarization_controller_thorlabs_mpc320_stub.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from pnpq.apt.protocol import ChanIdent, JogDirection
+from pnpq.devices.polarization_controller_thorlabs_mpc_stub import (
+    PolarizationControllerThorlabsMPC320Stub,
+)
+from pnpq.units import pnpq_ureg
+
+
+@pytest.fixture(name="stub_mpc")
+def stub_waveplate_fixture() -> PolarizationControllerThorlabsMPC320Stub:
```

Comment:
Return the abstract base class instead to make sure we are only testing methods defined on the ABC.